webpack.config.js

```diff
@@ -1,50 +1,48 @@
-var webpack = require('webpack');
-var WebpackBar = require('webpackbar');
-var Encore = require('@symfony/webpack-encore');
-var path = require('path');
-
-Encore
-
-  .addPlugin(new WebpackBar({
-    profile: Encore.isProduction() ? true:false,
+const webpack = require("webpack");
```

Comment:
I was working on these formatting changes in #237. I think I'd prefer `'` instead of `"`, like we do in our PHP code. 

(apart from that: All of our JS and SCSS files need some tidying up. :-) )

Reply:
this is what linting did. I'm not sure how to configure it.

Reply:
Ok, feel free to leave it like this, and i'll tweak ESlint / Prettier

 (or maybe Marcin can do that?)